prov/util/src/util_attr.c

```diff
@@ -314,6 +314,38 @@ int ofix_getinfo(uint32_t version, const char *node, const char *service,
 	return ret;
 }
 
+/* Caller should use only fabric_attr in returned core_info */
+int ofi_get_core_info_fabric(struct fi_fabric_attr *util_attr,
+			     struct fi_info **core_info)
```

Comment:
I thought about that. The only issue is it would require some extra copying for strings and adding one more function for freeing the fabric_attr. Are we okay with that?